### PR TITLE
minor qol improvement in debug outfit

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
minor qol improvement in debug outfit
debug outfit is faster and knows all surgeries, and can see ghosts.

It's funny that slowdown trait also prevents movement speed boosts

## Why It's Good For The Game
qol

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
tweak: debug outfit is faster and knows all surgeries, and can see ghosts.
/:cl:

### CODE REVIEW

**Comment by Archanial** [comment]
Speedup is just negative slowndown :trollface:

File: code/game/objects/items/debug_items.dm [hunk 1]
```diff
@@ -288,16 +287,19 @@
 		TRAIT_MEDICAL_HUD,
 		TRAIT_SECURITY_HUD,
 		TRAIT_BARMASTER,
+		TRAIT_ALL_SURGERIES,
 		TRAIT_METALANGUAGE_KEY_ALLOWED
 	)
 
-/obj/item/debug/orb_of_power/pickup(mob/user)
+/obj/item/debug/orb_of_power/pickup(mob/living/user)
```
Comment (Archanial) [inline code/game/objects/items/debug_items.dm]: Can not living mob pick this up?
Would be safer to just cast bellow

Comment (PowerfulBacon) [reply code/game/objects/items/debug_items.dm]: Only living mobs have inventory

**Review by PowerfulBacon — APPROVED** [review]

File: code/game/objects/items/debug_items.dm (hunk 1 above)
Comment (EvilDragonfiend) [reply code/game/objects/items/debug_items.dm]: well, changed anyway

**Review by PowerfulBacon — DISMISSED** [review]
Can we remove the stone making you unable to be converted to cult?

**Review by Archanial — APPROVED** [review]

File: code/game/objects/items/debug_items.dm [hunk 2]
```diff
@@ -297,13 +296,20 @@
...
 	var/datum/atom_hud/hud = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
 	hud.add_hud_to(user)
 	hud = GLOB.huds[DATA_HUD_SECURITY_ADVANCED]
 	hud.add_hud_to(user)
 	hud = GLOB.huds[DATA_HUD_DIAGNOSTIC_ADVANCED]
 	hud.add_hud_to(user)
+	
+	if(!isliving(user))
+		user.update_sight()
+		return .
```
Comment (Archanial) [inline code/game/objects/items/debug_items.dm]:
```suggestion
		return
```

**Pull request merged**